pysat/tests/test_meta.py

```diff
@@ -1119,6 +1222,8 @@ def test_get_attribute_name_case_preservation_w_higher_order_2(self):
                 == 'YoYoYO')
 
     def test_get_attribute_name_case_preservation_w_ho_reverse_order(self):
+        """Test that getting attribute names preserves the case when reversed.
+        """
```

Comment:
'reversed assignment order' ? ... don't think there is room.

Reply:
Will the higher order tests be removed as part of #789?  If so, we may not need to spend too much time wordsmithing these.